src/App.tsx

```diff
@@ -15,71 +15,101 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+type State = {
+  name: string,
+  clicked: boolean,
+};
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export class App extends React.Component<{}, State> {
+  state = {
+    name: '',
```

Comment:
`name: goods[goods.indexOf('Jam')] || goods[0],`

It's a more safe variant.

Reply:
fixed